tests/platform_tests/test_advanced_reboot.py

```diff
@@ -5,6 +5,7 @@
 from tests.common.fixtures.duthost_utils import backup_and_restore_config_db
 
 pytestmark = [
+    pytest.mark.sanity_check(skip_sanity=True),
```

Comment:
Why this change? Looks like the sanity will now be always skipped for every kind of advanced reboot test.
There is an option to skip the sanity from the command line arg `--skip_sanity`

Reply:
Removed skipping sanity check.